test/test_goodmans_er.py

```diff
@@ -1,6 +1,76 @@
 """Test Goodmans Ecological Regression."""
+import numpy as np
+import pytest
 from pyei.goodmans_er import *  # pylint:disable=wildcard-import,unused-wildcard-import
```

Comment:
I'd probably use explicit imports here instead of `*`. If all you are using is `GoodmansER`, you might even just `from pyei import GoodmansER`, which also tests that the import statement we expect users to use actually works

Reply:
Got it, will do!